src/main/scala/io/iohk/ethereum/network/handshaker/EtcHelloExchangeState.scala

```diff
@@ -26,19 +26,17 @@ case class EtcHelloExchangeState(handshakerConfiguration: EtcHandshakerConfigura
 
   override def applyResponseMessage: PartialFunction[Message, HandshakerState[PeerInfo]] = { case hello: Hello =>
     log.debug("Protocol handshake finished with peer ({})", hello)
-    if (
-      handshakerConfiguration.protocolVersion == ProtocolVersions.PV64 && hello.capabilities.contains(Etc64Capability)
-    )
-      EtcNodeStatus64ExchangeState(handshakerConfiguration)
-    else if (hello.capabilities.contains(Eth63Capability))
-      EtcNodeStatus63ExchangeState(handshakerConfiguration)
-    else {
-      log.debug(
-        s"Connected peer does not support eth ${ProtocolVersions.PV63.toByte} / ${ProtocolVersions.PV64.toByte} protocol. Disconnecting."
-      )
-      DisconnectedState(Disconnect.Reasons.IncompatibleP2pProtocolVersion)
+    protocolNegotiator.negotiate(hello.capabilities) match {
+      case Some(ProtocolVersions.PV64) =>
+        EtcNodeStatus64ExchangeState(handshakerConfiguration)
+      case Some(ProtocolVersions.PV63) =>
+        EtcNodeStatus63ExchangeState(handshakerConfiguration)
```

Comment:
For consistency: shouldn't it be called `EtcNodeStatus60ExchangeState`? PV60 is where this message comes from. Likewise, when 64 becomes 66 we will rename `EtcNodeStatus64ExchangeState` -> `EtcNodeStatus66ExchangeState`. But when 67, 68... come along we will not be renaming it.

(Yes, I am guilty of calling it 63 in previous versions of the code)